src/js_network_transport.hpp

```diff
@@ -168,7 +168,10 @@ struct JavaScriptNetworkTransport : public app::GenericNetworkTransport {
 
     using SendRequestHandler = void(ContextType m_ctx, const app::Request request, std::function<void(const app::Response)> completion_callback);
 
-    JavaScriptNetworkTransport(ContextType ctx) : m_ctx(ctx) {};
+    JavaScriptNetworkTransport(ContextType ctx) : m_ctx(ctx),
+        m_dispatcher { JavaScriptNetworkTransport::send_request_to_server_impl }
```

Comment:
The syntax here is misleading; the spaces around { } make it look like a block rather than a member intialization.

Reply:
👍  will fix